core/broadcast.go

```diff
@@ -118,24 +118,23 @@ func (b *echoBroadcast) BroadcastDKG(c context.Context, p *drand.DKGPacket) (*dr
 	addr := net.RemoteAddress(c)
 	dkgPacket, err := protoToDKGPacket(p.GetDkg())
 	if err != nil {
-		b.l.Debugw("", "echoBroadcast", "received invalid packet", "from", addr, "err", err)
+		b.l.Errorw("received invalid packet DKGPacket", "from", addr, "err", err)
 		return nil, errors.New("invalid packet")
 	}
 
 	hash := hash(dkgPacket.Hash())
 	if b.hashes.exists(hash) {
-		// if we already seen this one, no need to verify even because that
+		// if we've already seen this one, no need to verify even because that
 		// means we already broadcasted it
-		b.l.Debugw("", "echoBroadcast", "ignoring duplicate packet", "from", addr, "type", fmt.Sprintf("%T", dkgPacket))
+		b.l.Debugw("ignoring duplicate packet", "from", addr, "packet index", dkgPacket.Index(), "type", fmt.Sprintf("%T", dkgPacket))
 		return new(drand.Empty), nil
 	}
 	if err := b.verif(dkgPacket); err != nil {
-		b.l.Debugw("", "echoBroadcast", "received invalid signature", "from", addr)
+		b.l.Debugw("received invalid signature", "from", addr)
 		return nil, errors.New("invalid packet")
```

Comment:
mentioned invalid signature here too - maybe it should be `Error` in line with the other invalid packet